Duckling/Temperature/EN/Rules.hs

```diff
@@ -62,6 +62,19 @@ ruleTemperatureFahrenheit = Rule
       _ -> Nothing
   }
 
+ruleTemperatureKelvin :: Rule
+ruleTemperatureKelvin = Rule
+  { name = "<temp> Kelvin"
+  , pattern =
+    [ Predicate $ isValueOnly True
+    , regex "k(el?vi?n?s?)?\\.?"
```

Comment:
this regex is way too permissive. it will match:

kev
kelvs
kevi
kevns
kevin
kevins

just to throw out a few that make no sense. I suppose we want to match:

k
ks
kelvin
kelvins

which is `k(elvin)?s?`

Reply:
Thanks for the feedback. I was wondering about this. I updated it to the terse version.